### PR TITLE
stephanie's onboarding

### PR DESCRIPTION
**Pull request by cauliiflower**
# Strategy Onboarding

## TODO List

Onboarding tasks:

- [x ] Task 1: Implement `fibonacci_term()` in `fibonacci.py`
  - All tests in `test_fibonacci.py` should pass, run `pytest` command to check
- [x ] Task 2: Implement `find_min_index()` and `reverse_str_arr()`
  - Write tests that cover the logic that you wrote in `test_array.py`
- [x ] Task 3: Process and visualize sample data in `matplot.py`
  - Read data in CSV using `pandas` or `csv`, handle the missing values, plot using `matplotlib`
  - Include a screenshot of your graph in the PR

Some extra items:

- [ ] Completed [strategy welcome survey](https://forms.gle/SKozF7TUn8o385zd8)

### CODE REVIEW

File: onboarding/fibonacci.py [hunk 1]
```diff
@@ -14,5 +14,17 @@ def fibonacci_term(n):
     @returns integer, nth term of the Fibonacci sequence
     '''
 
-    # insert code here
-    pass
+    # checking valid inputs
+    if type(n) != int or n < 1:
+      return None
```
Comment (probro27) [inline onboarding/fibonacci.py:19]: raise TypeError if type is not int. Please check the test cases and do appropriately.

**Review by probro27 — CHANGES_REQUESTED** [review]
@cauliiflower Please check the review I have given. Also please attach the images produced by the matplotlib program. Thanks :)

**Comment by cauliiflower** [comment]
@probro27 Hi, this is a kinda stupid question, but I've updated my files and committed them onto my forked branch for my onboarding pr. How do I updated the files here? Thanks :))

**Comment by cauliiflower** [comment]
> @probro27 Hi, this is a kinda stupid question, but I've updated my files and committed them onto my forked branch/repository for my onboarding pr. How do I updated the files here? Thanks :))

Oops I think I committed to a new repository rather than my old one. How would I merge the two together or merge my new one with this pull request? Thanks.

Also, I didn't include graph 4 because it made my computer lag a bit when trying to process it.

**Comment by probro27** [comment]
@cauliiflower That's fine. Graph 4 is a bit weird. So I would suggest make the same changes in the prev repo since you can't really merge two repositories. 
If you are not able to do so, I can help you out over a call.

**Review by probro27 — APPROVED** [review]
@cauliiflower LGTM! Thanks for the PR!
We'll reach out to you for a one on one and your first ticket.
Welcome to the team!

**Pull request closed without merging**